src/passes/InstrumentMemory.cpp

```diff
@@ -309,6 +351,20 @@ struct InstrumentMemory : public WalkerPass<PostWalker<InstrumentMemory>> {
     curr->addFunction(std::move(import));
   }
 };
+struct InstrumentMemory : Pass {
+  // Adds calls to new imports.
+  bool addsEffects() override { return true; }
+
+  void run(Module* module) override {
+    auto arg = getArgumentOrDefault("instrument-memory", "");
+    InstructionFilter instructions = std::nullopt;
```

Comment:
```suggestion
    InstructionFilter instructions;
```
The default is nullopt.